packages/amplify-graphql-api-construct/API.md

```diff
@@ -364,7 +364,7 @@ export class SQLLambdaModelDataSourceStrategyFactory {
 export type SqlModelDataSourceDbConnectionConfig = SqlModelDataSourceSecretsManagerDbConnectionConfig | SqlModelDataSourceSsmDbConnectionConfig | SqlModelDataSourceSsmDbConnectionStringConfig;
 
 // @public
-export interface SqlModelDataSourceSecretsManagerDbConnectionConfig {
+export interface SqlModelDataSourceSecretsManagerDbConnectionConfig extends SslCertSsmPathAwareConfig {
```

Comment:
This is not ideal. When using secrets manager for credentials, I think we shouldn't use (mix) SSM for SSL certificate alone.

Reply:
We can 
(1) for now, support ssl certificate only when using SSM config (which is enough to support Gen2 DX).
(2) later, we will support secret manager for SSL certificate.

Reply:
Discussed offline. Will remove from Secrets Manager